docs/en/advanced/Namespace-Management.md

```diff
@@ -6,11 +6,28 @@ group: Advanced
 priority: 0
 ---
 
-TODO(David): Combine with Loading-Under-File-Storage-Metadata
-
 * Table of Contents
 {:toc}
 
+TODO: new outline
+
+Intro: Introduce concepts such as:
```

Comment:
should this intro part be a separate section?

Reply:
I think we can just have the introduction in the first block of text? (just like it is right now)